### PR TITLE
fix: imgStyle and imgClass in Image Component

### PR DESCRIPTION
**Pull request by jueinin**
1. width and height should only be applied on img element
parent element will auto-scale
2. add imgStyle and imgClass

close [#26996](https://github.com/ant-design/ant-design/issues/26996)

### CODE REVIEW

**Comment by jueinin** [comment]
@shaodahong I have changed the width and height props. could you review it?

File: src/Image.tsx [hunk 1]
```diff
@@ -47,6 +48,8 @@ const ImageInternal: React.FC<ImageProps> = ({
   onClick,
 
   // Img
+  imgClass,
+  imgStyle,
```
Comment (shaodahong) [inline src/Image.tsx:52]: default style or className set in `img`, and need `wrapperXXX`

Comment (jueinin) [reply src/Image.tsx]: ok, I have fixed it.

File: src/Image.tsx [hunk 2]
```diff
@@ -118,23 +121,27 @@ const ImageInternal: React.FC<ImageProps> = ({
...
         {...otherProps}
-        className={className}
+        className={wrappperClass}
         onClick={preview && !isError ? onPreview : onClick}
-        style={{
-          ...style,
-          width,
-          height,
-        }}
+        style={wrapperStyle}
```
Comment (shaodahong) [inline src/Image.tsx:144]: height and width should also set in the wrapper element

File: src/Image.tsx [hunk 3]
```diff
@@ -118,23 +121,27 @@ const ImageInternal: React.FC<ImageProps> = ({
...
+      `${prefixCls}-img`,
+      {
+        [`${prefixCls}-img-placeholder`]: placeholder === true,
+      },
+      className,
+    ),
+    style: {
+      height,
+      width,
+      ...style
```
Comment (shaodahong) [inline src/Image.tsx:134]:
```suggestion
    style: {
      height,
      ...style
```

Comment (shaodahong) [reply src/Image.tsx:134]: Keep as it is

File: src/Image.tsx (hunk 2 above)
Comment (shaodahong) [reply src/Image.tsx:144]: It's broken

Kapture 2020-10-16 at 16 54 12

File: src/Image.tsx (hunk 3 above)
Comment (jueinin) [reply src/Image.tsx]: sorry for the late reply. I have tested the performance in storybook. it should be OK.

**Review by shaodahong — APPROVED** [review]
LGTM

**Pull request merged**